webpack/common.js

```diff
@@ -13,8 +13,6 @@ module.exports = {
 		],
 	},
 	module: {
-		noParse: /node_modules\/vex-js\/dist\/js\/vex.js/,
```

Comment:
Vex doesn't appear to be referenced in our codebase any longer and `exprContextCritical` is deprecated.